### ISSUE TITLE
max_train_steps？

### ISSUE THREAD

**Issue by Gxt2002** [WU-CVGL/OmniScene#43]
Hello author! While reproducing your training, I noticed that you set max_train_steps = 5000, and the volume update stops at max_train_steps - 1000(i.e., 4000 steps). The visualization results show that the volume branch is far from being well-trained. However, when I tried to increase max_train_steps, gradient explosion occurred in the early training stages, likely because your learning rate is dynamically adjusted based on the number of iterations.
Could you please share what hyperparameters I should use to correctly reproduce the experiments? Would you mind sharing your configuration?
